docs/components/edge/edged.md

```diff
@@ -2,13 +2,13 @@
 
 ## Overview
 
-EdgeD is an edge node module which manages pod lifecycle. It helps user to deploy containerized workloads or applications at the edge node. Those workloads could perform any operation from simple telemetry data manipulation to analytics or ML inference and so on. Using `kubectl` command line interface at the cloud side, user can issue commands to launch the workloads.
+EdgeD is an edge node module which manages pod lifecycle. It helps users to deploy containerized workloads or applications at the edge node. Those workloads could perform any operation from simple telemetry data manipulation to analytics or ML inference and so on. Using `kubectl` command line interface at the cloud side, users can issue commands to launch the workloads.
 
-Docker container runtime is currently supported for container and image management. In future other runtime support shall be added, like containerd etc.,
+The Docker container runtime is currently supported for container and image management. In the future support for other runtimes shall be added, like containerd etc.,
```

Comment:
@daixiang0 @alanisaac , CRI has already been supported by edgecore, how about delete these line?

Reply:
Please update it refer to [doc](https://docs.kubeedge.io/en/latest/configuration/cri.html).

Reply:
fixed in 3db6642
included a reference to the avoce linked documentation on how to configure for other runtimes